src/main/java/gmail/mishchuk/modultest_2/PhoneBook.java

```diff
@@ -0,0 +1,64 @@
+package gmail.mishchuk.modultest_2;
+
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+public class PhoneBook {
+
+    private Map<String, String> contacts = new HashMap<>();
```

Comment:
Краще зробити `final`.

У нас є методи які оперують нам мапою `contacts` і ми створюємо мапу лише один раз.

```suggestion
    private final Map<String, String> contacts = new HashMap<>();
```